deliver/spec/upload_metadata_spec.rb

```diff
@@ -257,11 +261,17 @@ def create_metadata(path, text)
               description: { "en-US" => "App description" }
           }
 
-          # Get number of verions (used for if whats_new should be sent)
+          # Get number of versions (used for if whats_new should be sent)
           expect(Spaceship::ConnectAPI).to receive(:get_app_store_versions).and_return(app_store_versions)
 
           expect(version).to receive(:update).with(attributes: {})
 
+          # Get app info
+          expect(app_info).to receive(:get_app_info_localizations).and_return([app_info_localization_en])
+
+          # Get app info localization English (Used to compare with data to upload)
+          expect(app_info_localization_en).to receive(:name).and_return("")
```

Comment:
💭  `rubocop` might complain if you use double-quotes while you don't need any interpolation

```suggestion
          expect(app_info_localization_en).to receive(:name).and_return('')
```

💡  Also, might make it cleared and easier to understand the logic of the spec to provide a dummy name instead?

```suggestion
          expect(app_info_localization_en).to receive(:name).and_return('Old App Name')
```

Reply:
>💭  rubocop might complain if you use double-quotes while you don't need any interpolation

The rubocop rules we have in _fastlane_ explicitly allows both `"` and `'` to be used 😅

Reply:
TIL!